avro-converter/src/main/java/io/confluent/connect/avro/AvroConverter.java

```diff
@@ -102,8 +101,9 @@ public SchemaAndValue toConnectData(String topic, byte[] value) {
 
   private static class Serializer extends AbstractKafkaAvroSerializer {
 
-    public Serializer(SchemaRegistryClient client) {
+    public Serializer(SchemaRegistryClient client, boolean autoRegister) {
```

Comment:
`autoRegisterSchemas` might even be a better name here as well. also for the class field.